src/frontend/services/playlistServices.js

```diff
@@ -0,0 +1,39 @@
+import axios from "axios";
+
+export const getAllPlaylistsService = (token) => {
+  return axios.get("/api/user/playlists", {
+    headers: { authorization: token },
+  });
+};
+
+export const createPlaylistService = (token, playlistData) => {
+  return axios.post(
+    "/api/user/playlists",
+    { playlist: playlistData },
+    {
+      headers: { authorization: token },
+    }
+  );
+};
+
+export const addVideoToPlaylistService = (token, video, playlistId) => {
+  return axios.post(
+    `/api/user/playlists/${playlistId}`,
```

Comment:
You can put all the api address in constants. So that to avoide spelling mistake